package.json

```diff
@@ -2,7 +2,7 @@
   "name": "resin-supervisor",
   "version": "1.11.0",
   "scripts": {
-    "postinstall": "bash postinstall.sh",
+    "postinstall": "sh postinstall.sh",
```

Comment:
Can/should this be `./postinstall.sh`? So that it uses the hashbang of the script itself?